### PR TITLE
Created entities for profile and messenger

### PR DESCRIPTION
**Pull request by Mellifaro**

### CODE REVIEW

File: basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/Contact.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package com.softgroup.common.dao.api.entities;
+
+import javax.persistence.*;
+
+/**
+ * Created by Виктор on 03.04.2017.
+ */
+@Entity
+@Table(name = "contacts")
+public class Contact {
```
Comment (Dominys) [inline basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/Contact.java]: @Mellifaro Where is ID field ?

File: basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationMember.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package com.softgroup.common.dao.api.entities;
+
+import javax.persistence.*;
+
+/**
+ * Created by Виктор on 03.04.2017.
+ */
+@Entity
+@Table(name = "conversation_members")
+public class ConversationMember {
```
Comment (Dominys) [inline basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationMember.java]: @Mellifaro Where is ID field ?

File: basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationSettingsEntity.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
+package com.softgroup.common.dao.api.entities;
+
+import javax.persistence.*;
+
+/**
+ * Created by Виктор on 03.04.2017.
+ */
+@Entity
+@Table(name = "conversation_settings")
+public class ConversationSettingsEntity {
```
Comment (Dominys) [inline basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/ConversationSettingsEntity.java]: @Mellifaro same

File: frontend/rest/rest-impl/src/main/java/com/softgroup/rest/impl/controllers/AuthorizationController.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
+//package com.softgroup.rest.impl.controllers;
```
Comment (Dominys) [inline frontend/rest/rest-impl/src/main/java/com/softgroup/rest/impl/controllers/AuthorizationController.java]: @Mellifaro drop commented code

**Comment by Mellifaro** [comment]
Corrected mistakes with id, dropped commented code

**Review by Dominys — APPROVED** [review]

**Pull request merged**